src/main/kotlin/no/nav/amt/deltaker/bff/testdata/TestdataService.kt

```diff
@@ -0,0 +1,123 @@
+package no.nav.amt.deltaker.bff.testdata
+
+import kotlinx.coroutines.delay
+import no.nav.amt.deltaker.bff.deltaker.DeltakerService
+import no.nav.amt.deltaker.bff.deltaker.PameldingService
+import no.nav.amt.deltaker.bff.deltaker.forslag.kafka.ArrangorMeldingProducer
+import no.nav.amt.deltaker.bff.deltaker.model.Deltaker
+import no.nav.amt.deltaker.bff.deltaker.model.Pamelding
+import no.nav.amt.deltaker.bff.deltaker.model.Utkast
+import no.nav.amt.deltaker.bff.deltakerliste.Deltakerliste
+import no.nav.amt.deltaker.bff.deltakerliste.DeltakerlisteService
+import no.nav.amt.deltaker.bff.deltakerliste.tiltakstype.toInnhold
+import no.nav.amt.lib.models.arrangor.melding.EndringFraArrangor
+import no.nav.amt.lib.models.deltaker.Deltakelsesinnhold
+import no.nav.amt.lib.models.deltakerliste.tiltakstype.Tiltakstype
+import java.time.LocalDate
+import java.time.LocalDateTime
+import java.util.UUID
+
+const val TESTVEILEDER = "Z990098"
+const val TESTENHET = "0314"
+
+// Plutselig Lagsport
+const val TESTARRANGORANSATT = "fff9a665-cbde-4dbc-9ef9-deb8681a0d6f"
```

Comment:
Vet ikke om det var et konkret behov for å bruke spesifikke veiledere osv her, men vil potensielt kjøre mer forutsigbart om man bare henter ut "en eller annen" ansatt fra databasen. Sånn at man ikke må inn og endre her hvis veilederen eller enheten av en eller annen grunn ikke skulle finnes mer.